README.md

```diff
@@ -1,2 +1,9 @@
 # alacritty
 An ambitious theme for alacritty
+
+![Screenshot](./screenshot.png)
+
+## How to use
+
+Edit alacritty configuration file (usually located at `$XDG_CONFIG_HOME/kitty/kitty.conf`) and copy embark to your
```

Comment:
Just need to update this path for alacritty

Reply:
:facepalm: done